public/templates/header.php

```diff
@@ -18,6 +18,8 @@
     <script src="//code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 
     <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/email.css">
+    <link rel="stylesheet" href="css/table_style.css">
```

Comment:
these css files aren't exactly relevant to all of our files right?

Reply:
i think we had agreed to keep those in the header at some point in the past

Reply:
yeah but loading stylesheets on pages where we don't need them is just not good, one other downside is that one would have to be very careful with classes and IDs because styling that is not meant for a particular element maybe applied on it anyway because of a naming overlap